nexus/db-lookup/src/lookup.rs

```diff
@@ -909,6 +923,15 @@ lookup_resource! {
     ]
 }
 
+lookup_resource! {
+    name = "ScimClientBearerToken",
+    ancestors = ["Silo"],
+    lookup_by_name = false,
+    soft_deletes = true,
+    primary_key_columns = [ { column_name = "id", rust_type = Uuid } ],
+    visible_outside_silo = true // XXX needed?
```

Comment:
I think it is needed, otherwise the authz check for a fleet viewer/collab/admin who is not in the silo in question will fail.